packages/editor-ui/src/utils/htmlUtils.ts

```diff
@@ -37,6 +37,13 @@ export function sanitizeHtml(dirtyHtml: string) {
 	return sanitizedHtml;
 }
 
+export const sanitizeInput = <T>(message: T): string | T => {
```

Comment:
Maybe this could be named `sanitizeInputIfNeeded` or `sanitizeIfString` to indicate that it conditionally sanitizes the input. Also a JSDoc comment would be nice (e.g. what does sanitize mean in this case)  🙏

Reply:
@tomi done! let me know if you had something different in mind